parley/src/tests/utils/renderer.rs

```diff
@@ -19,7 +19,15 @@ use tiny_skia::{Color, FillRule, Paint, PathBuilder, Pixmap, PixmapMut, Rect, Tr
 
 #[derive(Clone, Copy, Debug, PartialEq)]
 pub(crate) struct ColorBrush {
-    pub(crate) color: Color,
+    pub(super) color: Color,
+}
+
+impl ColorBrush {
+    pub(crate) fn new(r: u8, g: u8, b: u8, a: u8) -> Self {
```

Comment:
Rather just see this take a `Color` argument and then you can pass in things from `color::palette::css` instead which also gives them better names than a series of constants.

Reply:
Didn't know those were available--done